documentation/api/query/v4/index.markdown

```diff
@@ -30,6 +30,9 @@ operator or a [PQL][pql] query string.
 a query with a [`from`][from] is required to choose the [entity][entities] for which to query. For
 general info about queries, see [our guide to query structure.][query]
 
+* `parse_pql_only`: optional. A boolean value. When true, the query response 
```

Comment:
what does this do for AST? Wondering if having "pql" in the name is appropriate.